.github/workflows/unit-testing-js.yaml

```diff
@@ -0,0 +1,19 @@
+name: Unit Tests on JavaScript
```

Comment:
I think this can also be part of the `reusable-verification`, since we also run linters and tests after each-other in case of golang as part of the verification.

Reply:
Sure, let's do that.